modules/tinymce/src/core/test/ts/browser/keyboard/SpaceKeyTest.ts

```diff
@@ -16,12 +16,19 @@ describe('browser.tinymce.core.keyboard.SpaceKeyTest', () => {
 
   context('Space key around inline boundary elements', () => {
     it('Press space at beginning of inline boundary inserting nbsp', () => {
+      let inputEvents = 0;
       const editor = hook.editor();
       editor.setContent('<p>a <a href="#">b</a> c</p>');
+      editor.on('beforeinput input', (e) => {
+        // eslint-disable-next-line no-console
+        console.log(e.isComposing);
+        inputEvents++;
```

Comment:
Better to use an array to capture e.type and assert that since this could be firing `input` twice and still work.